precompiled/serde_derive/Cargo.toml

```
@@ -17,6 +17,7 @@ rust-version = "1.56"
 [features]
 default = []
 deserialize_in_place = []
+compiled = ["proc-macro2", "quote", "syn"]
```

```suggestion
from_source = ["proc-macro2", "quote", "syn"]
```

I think compiled would imply that it's opt-in but given we are opt-out it would perhaps work as a name ?